lib/util/http/github.ts

```diff
@@ -493,4 +493,24 @@ export class GithubHttp extends Http<GithubHttpOptions> {
 
     return result;
   }
+
+  public getRawFile(
+    url: string,
+    options: InternalHttpOptions & GithubHttpOptions = {},
+  ): Promise<HttpResponse> {
+    const newOptions: InternalHttpOptions & GithubHttpOptions = {
+      ...options,
+      headers: {
+        accept: 'application/vnd.github.raw+json',
+      },
+    };
+
+    let newURL = url;
+    const httpRegex = regEx(/^https?:\/\//);
+    if (options.repository && !httpRegex.test(options.repository)) {
+      newURL = joinUrlParts(options.repository, 'contents', url);
+    }
```

Comment:
why do we need this? only baseUrl should be added if url isn't absolute. 🤔

the repo field is used for auth only, see request method

Reply:
Or do you simply mean `repository` should be removed from `newOptions` so it is not fed into `request`?

Reply:
No, my idea with relative paths where to allow `foo/bar/contents/lore/ipsum.txt` urls, which should just work

Reply:
Yeah, they work already. I have added some tests to prevent regressions.